docs/daemon/network.mdx

```diff
@@ -0,0 +1,169 @@
+---
+sidebar_position: 4
+sidebar_label: "Testnet"
+id: daemon-network-setup
+title: Network Setup
+description: Deploying a Kwil Network with multiple nodes
+slug: /daemon/network/setup
+---
+
+This guide provides step-by-step instructions for configuring and deploying a Kwil network with an initial set of M validator and N non-validator nodes. 
+
+## Network Configuration
+The `setup testnet` command can be used to generate the configuration required for multiple nodes, all with the same genesis configuration, and pre-configured to connect to each other. Refer to the [testnet deployment guide](./../admin/setup#creating-a-new-test-network-of-several-nodes) for more information on the command usage. 
+
+```bash title="Example"
+# Generate configuration required for a testnet corresponding to 2 validator nodes and 1 non-validator nodes with IP addresses starting from '192.168.10.10' at location ~/.kwild
+
+$ kwil-admin setup testnet -v 2 -n 1 -o "~/.kwild" --starting-ip '192.168.10.10'
+Successfully initialized 3 node directories: ~/.kwild
+
+# Running the above command generates 3 directories node0, node1, node2 under ~/.kwild. 
+# Each node directory includes corresponding genesis.json, config.toml, private_key configuration files as shown below
+
+$ ls ~/.kwild                                                               
+node0   node1   node2
+
+$ tree ~/.kwild/node0
+~/.kwild/node0
+├── abci
+│   ├── config
+│   │   └── genesis.json
+│   └── data
+├── config.toml
+└── private_key
+
+4 directories, 3 files
+
+# The persistent_peers config in the config.toml is populated based on the IP addresses configured using either the `--hostname`  or the `--starting-ip-address` flag 
+$ cat ~/.kwild/node0/config.toml| grep persistent_peers
```

Comment:
https://en.wikipedia.org/wiki/Cat_%28Unix%29#Useless_use_of_cat (the grep example)
:)